src/main/java/com/jcraft/jsch/Log4j2Logger.java

```diff
@@ -19,6 +19,15 @@ public void log(int level, String message) {
     logger.log(getLevel(level), message);
   }
 
+  @Override
+  public void log(int level, String message, Throwable cause) {
+    if (cause != null) {
+      logger.log(getLevel(level), message);
+      return;
+    }
+    logger.log(getLevel(level), message, cause);
+  }
+
```

Comment:
Since we added an `import` statement for the `Logger` class for the other implementations, we should probably do the same thing for this class now.

Reply:
I think I've changed that already without seeing your comment, haven't I?